docs/connector-development/tutorials/the-hard-way/build-a-connector-the-hard-way.md

```diff
@@ -20,7 +20,7 @@ demonstrate the following concepts in action:
 **This tutorial is meant for those interested in learning how the Airbyte Specification
 works in detail, not for creating production connectors**. If you're building a real source, you
 should start with using the [Connector Builder](../../connector-builder-ui/overview), or the
-[Connector Development Kit](https://github.com/airbytehq/airbyte/tree/master/airbyte-cdk/python/docs/tutorials).
+[Connector Development Kit](../../cdk-python/).
```

Comment:
This I assume links to a docs page, not the repo? I think it's okay to link to the repo too if this breaks in weird ways.